run_tests.sh

```diff
@@ -1 +1 @@
-nosetests --with-coverage  --cover-package=ease
\ No newline at end of file
+pytest --cov=ease
```

Comment:
Could you please do 2 things
1. Remove run_tests.sh if it is not being used in this app
2. Add branch coverage to pytest

Reply:
sure